scripts/release.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+atlas_api="https://atlas.hashicorp.com/api/v1/box/hashbang/shell-server"
+atlas_token=$(jq -r '.atlas_token' config.json)
+bintray_user=$(jq -r '.bintray_user' config.json)
+bintray_key=$(jq -r '.bintray_key' config.json)
+version="$VERSION"
```

Comment:
It should (tested on `sh` on Ubuntu 16.04) echo the line "Version number: " to the TTY and then read a single line from input, but only if `$VERSION` doesn't exist.